data/GMSim.py

```diff
@@ -0,0 +1,434 @@
+from re import T
+import numpy as np
+from scipy.stats import gamma
+import sys
+# import csv
+np.set_printoptions(threshold=sys.maxsize)
+
+
+def RJB2REPI(M, R):
```

Comment:
Can you please change these comments to into docstrings? This will help users query them later. Its just a simple change that goes like this (see help on docstrings [here](https://realpython.com/documenting-python-code/)): 

```
class SimpleClass:
    """Class docstrings go here."""

    def say_hello(self, name: str):
        """Class method docstrings go here."""

        print(f'Hello {name}')
```

Reply:
Just the function definition comments, right?

Reply:
Yes, that's right.